submissions/BlessingOkoro/ecommerce-project/tests/Application/Actions/User/ListUserActionTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
```

Comment:
not needed

Reply:
Everything in tests folder actually should go. Unless you wrote tests for your api?